shared/middleware/helpers/animation_bucket.rb

```diff
@@ -48,4 +48,12 @@ def s3_get_object(key, if_modified_since, version)
     )
     s3_object
   end
+
+  # Helper class that returns the latest non-deleted version of the key
```

Comment:
Nit: This is a helper _method_, not a helper _class_.  It would probably be good style to make this helper private too.